commands/move.go

```diff
@@ -2,50 +2,49 @@ package commands
 
 import (
 	"fmt"
-	"os"
 
+	"github.com/bndw/pick/errors"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 func init() {
 	rootCmd.AddCommand(&cobra.Command{
 		Use:   "mv [name] [new-name]",
 		Short: "Rename a credential",
-		Long: `The move command is used to rename a credential.
-            `,
+		Long:  "The move command is used to rename a credential.",
 		Run: func(cmd *cobra.Command, args []string) {
-			os.Exit(Move(args...))
+			runCommand(Move, cmd, args)
 		},
 	})
 }
 
-func Move(args ...string) int {
+func Move(args []string, flags *pflag.FlagSet) error {
 	safe, err := newSafeLoader().Load()
 	if err != nil {
-		return handleError(err)
+		return err
```

Comment:
We should use `github.com/pkg/errors.Wrap` here, but this applies to the whole codebase and thus needs to be done in a different PR.

Reply:
Agreed